docs/introduction/record_agent.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: "contents"
+title: Recording Agents
+---
+
+# Recording Agents
+
+```{eval-rst}
+ .. py:currentmodule: gymnasium.wrappers
+
+During and after training an agnt, it is interesting to record how the agent acts over an episode, in particular to video the agent behaviour and log the total reward for each episode. This can be achieved through two wrappers: :class:`RecordEpisodeStatistics` and :class:`RecordVideo`, the first will track episode data such as the total rewards, episode length and time taken with the second saving the environment rendering as mp4 videos.
+
+We consider how to apply these wrappers for two types of problems; the first for recording data for every episode and second for recording data periodiclly.
+```
+
+## Recording Every Episode
+
+Given a trained agent, when evaluating an agent, you would wish to record several episodes to see how the agent acts.
```

Comment:
Given a trained agent, you may wish to record several episodes during evaluation to see how the agent acts.